docs/observation/payload.md

```diff
@@ -1,34 +1,52 @@
 # Format of observation payload
 
-Notifications received by a source repository, such as GitHub, OneDrive, Google Drive, etc. have a provider specific format, e.g. OneDrive uses the format described on Microsoft's [docs](https://docs.microsoft.com/en-us/graph/api/driveitem-delta?view=graph-rest-1.0&tabs=http) site. When those notifications are received in the `onedrive-change-listener`, they get converted to a portable format before sent to the next stage, e.g.:
+Notifications received by a source repository, such as GitHub, OneDrive, Google Drive, etc. have a provider specific format, e.g. OneDrive uses the format described on Microsoft's [docs](https://docs.microsoft.com/en-us/graph/api/driveitem-delta?view=graph-rest-1.0&tabs=http) site. When those notifications are received in the respective listener (e.g. `onedrive-change-listener` for OneDrive), they get converted to a portable format before sent to the next stage, e.g.:
 
 ```
 {
   "type": "onedrive",
-  "provider" {
```

Comment:
why did you remove this?

Reply:
It contained the `driveId` only. Since the `sourceHash` (or `uid`) is now part of the payload, the `driveId` is no longer required for calculation, do you still find it necessary to be included?

Reply:
well, maybe another consumer of the events might be interested in the drive id.